FrameworkSystem/Client/ComponentInstaller.py

```diff
@@ -2085,20 +2076,6 @@ def installPortal(self):
         DIRAC.exit(-1)
       return S_ERROR(error)
 
-    # compile the JS code
-    prodMode = ""
-    webappCompileScript = os.path.join(self.linkedRootPath, "WebAppDIRAC/scripts", "dirac-webapp-compile.py")
-    if os.path.isfile(webappCompileScript):
-      os.chmod(webappCompileScript, self.gDefaultPerms)
-      gLogger.notice("Executing %s..." % webappCompileScript)
-      if os.system("python '%s' > '%s.out' 2> '%s.err'" % (webappCompileScript,
-                                                           webappCompileScript,
-                                                           webappCompileScript)):
-        gLogger.error("Compile script %s failed. Check %s.err" % (webappCompileScript,
-                                                                  webappCompileScript))
-      else:
-        prodMode = "-p"
-
```

Comment:
@zmathe I actually think the changes I have done in this file should be backported...?

Reply:
I do not think. The users who does not use the new management tools, they can still able to compile WebApp in 'old' way. In some point they will have to move to use the management tools.